virttest/shared/scripts/virtio_console_guest.py

```diff
@@ -113,7 +113,7 @@ def blocking(self, port, mode=False):
         """
         raise NotImplementedError
 
-    def async(self, port, mode=True, exp_val=0):
+    def asynchronous(self, port, mode=True, exp_val=0):
```

Comment:
This change requires at least to change the `io-github-autotest-qemu/qemu/tests/virtio_console.py` as well, not sure if other tests as well...

Reply:
Good point! Here it is: https://github.com/autotest/tp-qemu/pull/2485